composer.json

```diff
@@ -20,7 +20,10 @@
     "autoload": {
         "psr-4": {
             "PackageVersions\\": "src/PackageVersions"
-        }
+        },
+        "files": [
```

Comment:
using ``files`` is quite bad, because this forces loading the file all the time instead of using autoloading